src/database/migrations/20230210023333_create_coaching_styles_table.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return await knex.schema.createTable("styles", (t) => {
```

Comment:
would it make more sense for this to be `coaching_styles`?

Reply:
Yeah I made it that because of the `coaches_styles` join table